ci-operator/config/openshift/installer/openshift-installer-release-4.3.yaml

```diff
@@ -7,7 +7,7 @@ build_root:
   image_stream_tag:
     name: release
     namespace: openshift
-    tag: golang-1.12
+    tag: golang-1.13
```

Comment:
let's not do the 4.3, only 4.4 and master

Reply:
> let's not do the 4.3, only 4.4 and master

Dropped 4.3 with 0af6b8aa0 -> e8ad93b70.  Kept FCOS based on [this][1], but I can drop that too if folks want.

[1]: https://github.com/openshift/release/pull/5928#issuecomment-565529743